src/wwwroot/styles/views/_settings.scss

```diff
@@ -656,19 +656,29 @@
     ol{
       padding-left: 15px;
       margin-bottom: 0px;
+      color: $suit-accent-color;
       li{
         counter-increment: list;
         position: relative;
         text-align: left;
+        padding-top: 5px; 
```

Comment:
is this necessary?

Reply:
No you don't, padding i'ts not the right call here, use margin instead -> https://stackoverflow.com/questions/2189452/when-to-use-margin-vs-padding-in-css